extra/docs_landing.html

```diff
@@ -1 +1,128 @@
+                        <li class="listitem"><a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/example-using-index-lifecycle-policy.html">Manage data retention</a></li>
+                        <li class="listitem"><a href="https://www.elastic.co/guide/en/kibana/current/dashboard.html">Visualize data with dashboards</a></li>
+                    </ul>
+                </div>
+            </td>
+            <td>
+                <div class="itemizedlist">
+                    <ul class="itemizedlist" type="disc">
+                        <li class="listitem"><a href="https://www.elastic.co/guide/en/cloud/current/ec-billing-models.html">Elastic Cloud</a></li>
+                        <li class="listitem"><a href="https://aws.amazon.com/marketplace/pp/prodview-voru33wi6xs7k">AWS Marketplace</a></li>
+                        <li class="listitem"><a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/elastic.ec-azure-pp">Azure Marketplace</a></li>
+                        <li class="listitem"><a href="https://console.cloud.google.com/marketplace/product/endpoints/elasticsearch-service.gcpmarketplace.elastic.co">GCP Marketplace</a></li>
+                        <li class="listitem"><a href="https://cloud.elastic.co/pricing">Review pricing using our calculator</a></li>
+                    </ul>
+                </div>
+            </td>
+            </tr>
+        </table>
+
+        <h3>Need help?</h3>
+        <table width="100%">
+            <tr>
+                <td>
+                    <div class="itemizedlist">
+                        <ul class="itemizedlist" type="disc">
+                            <li class="listitem"><a href="https://cloud.elastic.co/support/">Support Portal</a></li>
+                            <li class="listitem"><a href="get-support-help.html">How to get support</a></li>
+                            <li class="listitem"><a href="elastic.co/guide/index.html">Documentation</a></li>
+                            <li class="listitem"><a href="https://discuss.elastic.co/">Forums</a></li>
+
+                        </ul>
+                    </div>
+                </td>
+                <td>
+                    <div class="itemizedlist">
+                        <ul class="itemizedlist" type="disc">
+                            <li class="listitem"><a href="https://www.elastic.co/training/">Training</a></li>
+                            <li class="listitem"><a href="https://www.elastic.co/events/?tab=1&solution=null&event=Meetup&region=null&language=English&industry=null">Meetups and virtual events</a></li>
+                            <li class="listitem"><a href="https://www.elastic.co/videos/">Webinars</a></li>
+                            <li class="listitem"><a href="https://ela.st/slack">Community Slack</a></li>
+                        </ul>
+                    </div>
+                </td>
+            </tr>
+        </table>
+
+  <h3>All topics</h3>
```

Comment:
This heading can likely go. (A horizontal rule might be a good way to separate the welcome page from the link farm.)